src/Libraries/RevitNodes/Elements/Views/ScheduleView.cs

```diff
@@ -249,6 +258,62 @@ public ScheduleView AddFields(List<Revit.Schedules.SchedulableField> fields)
         }
 
         /// <summary>
+        /// Add Schedule Filters to Schedule View.
+        /// </summary>
+        /// <param name="scheduleFilters">List of Schedule Filters.</param>
+        /// <returns name="scheduleView">Schedule View.</returns>
+        public ScheduleView AddFilters(List<ScheduleFilter> scheduleFilters)
+        {
+            var doc = DocumentManager.Instance.CurrentDBDocument;
+            TransactionManager.Instance.EnsureInTransaction(doc);
+
+            var currentFilters = this.InternalScheduleFilters.Select(x => new ScheduleFilter(x)).ToList();
+            foreach (var sf in scheduleFilters)
```

Comment:
nope, this is fine, my only concern was the nested contains check, but since the number is tiny it doesn't matter.